README.md

```diff
@@ -9,21 +9,49 @@ __i40-aas__ welcomes [contributions](#contributing). Please read about [known is
 
 
 ## Contents
-- [Description](#description)
-- [Prerequisites](#prerequisites)
-- [Instructions](#instructions)
-  - [Getting Started](#getting-started)
-  - [Cleanup](#cleanup)
-- [Configuration](#configuration)
-- [Known Issues](#known-issues)
-- [Get Support](#get-support)
-- [Contributing](#contributing)
-- [Upcoming Changes](#upcoming-changes)
-- [License](#license)
+- [i40-aas](#i40-aas)
+  - [Contents](#Contents)
+  - [Main Features](#Main-Features)
+  - [Context - Industry 4.0](#Context---Industry-40)
+  - [Prerequisites](#Prerequisites)
+  - [Instructions](#Instructions)
+    - [Getting Started](#Getting-Started)
+    - [Cleanup](#Cleanup)
+  - [Configuration](#Configuration)
+  - [Known Issues](#Known-Issues)
+  - [Get Support](#Get-Support)
+  - [Contributing](#Contributing)
+  - [Upcoming Changes](#Upcoming-Changes)
+  - [License](#License)
 
 
+## Main Features
+The AAS-Service provides an implementation of the RAMI 4.0 reference  architecture as specified by [Plattform Industrie 4.0](https://www.zvei.org/themen/industrie-40/details-of-the-asset-administration-shell/). Main features include:
+
+- Standardized and secure communication interfaces and API to access Asset Administration Shells of the digital representation of a physical asset (Digital Twin), that enable:
+  - use of a single AAS information model
+  - secure access to asset information from cloud and edge
+  - integration of assets without a communication interface (i.e. passive), e.g. via barcodes or QR-codes pointing to asset       management solutions
+
+- Facilitate Interoperability:
+  - The AAS-Service can act as a mediator between assets and applications that use different languages/standards
+  - Enable interactions between administration shells based on semantic protocols [(VDI/VDE 2193-1)](https://www.smart-production.de/open-automation/news-detailansicht/nsctrl/detail/News/vdivde-2193-einheitliche-sprache-fuer-industrie-40-komponenten-2019995/)
```

Comment:
use the original source.
https://www.vdi.de/richtlinien/details/vdivde-2193-blatt-1-sprache-fuer-i40-komponenten-struktur-von-nachrichten

Reply:
done